# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/LinkStyle.h
# Commit: 110862c2c4da
# Full Hash: 110862c2c4da9f9cb2b6ecc64668fac0b6275350
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2022-06-28 09:16:40
# Regressor Bug: 1774175
# File Overlap Count: 1
# Description:
#   Bug 1774175 - Call SetAcquiringImportMaps(false) when processing modulepreload. r=yulia,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149371
# ==============================================================================

diff -r 2841a1ebc685 -r 110862c2c4da dom/base/LinkStyle.h
--- a/dom/base/LinkStyle.h	Mon Jun 27 21:19:51 2022 +0000
+++ b/dom/base/LinkStyle.h	Mon Jun 27 21:20:48 2022 +0000
@@ -108,7 +108,8 @@
     eALTERNATE = 0x00000010,
     ePRECONNECT = 0x00000020,
     // NOTE: 0x40 is unused
-    ePRELOAD = 0x00000080
+    ePRELOAD = 0x00000080,
+    eMODULE_PRELOAD = 0x00000100
   };
 
   // The return value is a bitwise or of 0 or more RelValues.